controllers/authController.js

```diff
@@ -0,0 +1,32 @@
+const authService = require("../services/authService");
+const axios = require("axios");
+const error = require("../middlewares/errorConstructor");
+
+const kakaoSignIn = (req, res) => {
+  const headers = req.headers["authorization"];
+  const accessToken = headers.split(" ")[1];
+
+  axios.get("https://kapi.kakao.com/v2/user/me", {
+      headers: {
+        Authorization:
+          `Bearer ${accessToken}`,
+      },
+    }).then((result) => {
+      const nickname = result.data.kakao_account.profile.nickname;
+      const email = result.data.kakao_account.email;
+      const kakaoId = result.data.id.toString();
```

Comment:
데이터베이스 컬럼을 string -> int로 변경해주세요~
```suggestion
      const kakaoId = result.data.id;
```

Reply:
kakao측에서 제공하는 id가 int 자료형 범위를 초과하기 때문에 공식적으로 long타입을 권장하고 있어 int에서 bigint로 자료형을 변경하였습니다.